compose-builder/add-security.yml

```diff
@@ -216,6 +216,7 @@ services:
       KONGURL_SERVER: kong
       SECRETSTORE_HOST: edgex-vault
       SECRETSTORE_TOKENPATH: /tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json
+      EDGEX_SECURITY_SECRET_STORE: "true"
```

Comment:
If no longer using SecretStore (per https://github.com/edgexfoundry/edgex-go/pull/3205) why do we need `EDGEX_SECURITY_SECRET_STORE: "true"`?


Also `SECRETSTORE_HOST` and `SECRETSTORE_TOKENPATH` should be removed.

Reply:
ok, that makes sense, I'll close this PR then